pdh_check/README.md

```diff
@@ -43,3 +45,4 @@ The PDH check does not include any service checks.
 [5]: https://docs.datadoghq.com/agent/guide/agent-commands/#agent-status-and-information
 [6]: https://docs.datadoghq.com/developers/metrics/custom_metrics/
 [7]: https://docs.datadoghq.com/account_management/billing/custom_metrics/
+[8]: https://docs.datadoghq.com/integrations/windows_performance_counters/
```

Comment:
@ofek you know why this is not visible if the PR has been merged into integrations-core?